sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/view.scala

```diff
@@ -35,6 +35,56 @@ object EliminateView extends Rule[LogicalPlan] with CastSupport {
   }
 }
 
+/**
+ * ViewBindingMode is used to specify the expected schema binding mode when we want to create or
+ * replace a view in [[CreateViewStatement]].
+ */
+sealed trait ViewSchemaMode {
+  override val toString: String = getClass.getSimpleName.stripSuffix("$")
+}
+
+/**
+ * SchemaBinding means the view only tolerates minimal changes to the underlying schema.
+ * It can tolerate extra columns in SELECT * and upcast to more generic types.
+ */
+object SchemaBinding extends ViewSchemaMode {
+  override val toString: String = "BINDING"
```

Comment:
```suggestion
  override def toString: String = "BINDING"
```
since the function body is just a string literal